Source/Magick.NET/Shared/MagickImage.cs

```diff
@@ -4624,10 +4624,35 @@ public void Ping(byte[] data)
         /// <param name="readSettings">The settings to use when reading the image.</param>
         /// <exception cref="MagickException">Thrown when an error is raised by ImageMagick.</exception>
         public void Ping(byte[] data, MagickReadSettings readSettings)
+        {
+            Ping(data, 0, data.Length, readSettings);
+        }
+
+        /// <summary>
+        /// Reads only metadata and not the pixel data.
+        /// </summary>
+        /// <param name="data">The byte array to read the image data from.</param>
+        /// <param name="offset">The offset at which to begin reading data.</param>
+        /// <param name="count">The maximum number of bytes to read.</param>
+        /// <exception cref="MagickException">Thrown when an error is raised by ImageMagick.</exception>
+        public void Ping(byte[] data, int offset, int count)
```

Comment:
Could you also add some unit tests for this method? You can find some examples of how I did that with the `Read` method in the file `TheReadMethod.cs` in the `MagickImageTest` folder. It would be okay to just add tests for the new overloads that you created. But feel free to create a separate PR to add the unit tests for the `Ping` method first. Most of the `Read` overloads are also available as a `Ping` overload so this can probably be a copy paste "party". And if you prefer to let me add these overloads I would not mind to do that also.

Reply:
I'll look into it. But not now.